README.md

```diff
@@ -197,6 +198,32 @@ You can switch to `regex` mode dynamically by pressing `Ctrl-R` (Rotate Mode).
 | 1         | No Match found                      |
 | 130       | Aborted by Ctrl-C/Ctrl-G/ESC/etc... |
 
+# Tools compatible with `skim`
+
+These tools are or aim to be compatible with `skim`:
+
+## [fzf-lua neovim plugin](https://github.com/ibhagwan/fzf-lua)
+
+Install it with your package manager, following the README. For instance, with `lazy.nvim`:
+
+```lua
+{
+  "ibhagwan/fzf-lua",
+  -- optional for icon support
+  dependencies = { "nvim-tree/nvim-web-devicons" },
+  -- enable `sk` support instead of the default `fzf`
+  opts = {'skim'}
+}
+```
+
+## (wip) [nu_plugin_skim for nushell](https://github.com/idanarye/nu_plugin_skim)
```

Comment:
1. I don't think it needs to be marked as `(wip)`. As far as I consider it's in a perfectly usable state.
2. Do you think there should also be a link here to [the Nushell website](https://www.nushell.sh/), or is it enough if users who don't know what Nushell is will click on the link to get to my repository and get to the Nushell website from there?

Reply:
Sorry, I was looking at the first issue you opened that mentioned it being pre-release without realizing that it was 2 years old 😅